php_version_resolver.go

```diff
@@ -0,0 +1,81 @@
+package composer
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/paketo-buildpacks/packit/v2/fs"
+)
+
+type PhpVersionResolver struct{}
+
+func NewPhpVersionResolver() PhpVersionResolver {
+	return PhpVersionResolver{}
+}
+
```

Comment:
nit: can we get a comment explaining what this is doing?

Reply:
done